.github/workflows/ci_tests.yml

```
@@ -0,0 +1,227 @@
+name: GEOS CI 
+on: pull_request
+
+# Cancels in-progress workflows for a PR when updated
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  GEOSX_TPL_TAG: 220-932
+
+jobs:
+  # Matrix jobs will be cancelled if PR is a draft.
+  # PR status must be "Open" to run CI.
+  check_pull_request_is_not_a_draft:
+    # We use the most recent ubuntu distribution available in Github Actions to ensure maximum support of google cloud's sdk.
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check that the PR is not a draft (cancel rest of jobs otherwise)
+      run: |
+        echo "Is PR a draft?"
+        echo ${{ toJSON(github.event.pull_request.draft) }}
+        if [[ ${{ toJSON(github.event.pull_request.draft) }} == true ]]; then "false" ; else "true"; fi
+
+  # PR must be assigned to be merged.
+  # This job will fail if this is not the case.
+  check_pull_request_is_assigned:
+    needs: [check_pull_request_is_not_a_draft]
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```
Please change the `ubuntu-latest`: It's a not that it matters a lot, but I think it's better if we keep the exact same system everytime.